spec.json

```diff
@@ -344,6 +346,7 @@
         },
         "largeMotor": {
             "friendlyName": "Large Motor",
+            "systemDeviceNameConvention": "motor{0}",
```

Comment:
Not really related to this PR, but looks like we have a mix of two mask styles: `*` and `{0}`. May be we need to choose one.